examples/ASDKgram/Sample/CommentFeedModel.h

```diff
@@ -0,0 +1,27 @@
+//
+//  CommentFeedModel.h
+//  ASDKgram
+//
+//  Created by Hannah Troisi on 3/9/16.
+//  Copyright © 2016 Hannah Troisi. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```

Comment:
Minor, but maybe put this and all your UIKit.h includes into the precompiled header / .pch